js/DataManager/Data-Binding.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data-Binding
-description: data binding
+title: Data-Binding in JavaScript DataManager | Syncfusion
+description: Learn here about getting started with Syncfusion JavaScript DataManager control, its elements, and more.
```

Comment:
Change `data binding` instead of getting started

Reply:
resolved